src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String FIRST_SYMBOL = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        BufferedReader bufferedReader = null;
```

Comment:
let's declare and initialize the reader in try with resources

Reply:
done